src/runtime_src/core/common/api/xrt_module.cpp

```diff
@@ -914,7 +914,7 @@ class module_elf_aie2p : public module_elf
       
       uint32_t index = get_section_name_index(name);
       buf.append_section_data(sec.get());
-      map.emplace(std::make_pair(index, std::make_pair(sec_index, buf)));
+      map.emplace(index, std::pair{sec_index, buf});
```

Comment:
Ah yes,  std::pair is necessary here.  std::make_pair would have worked in this case, because the first and second of the pair are values, but emplacing as a tuple is good.